src/server/req_delete.c

```diff
@@ -327,6 +329,7 @@ req_deletejob(struct batch_request *preq)
 	int delhist = 0;
 	int maxindex = 0;
 	int count = 0;
+	extern job  *chk_job_request(char *, struct batch_request *, int *);
```

Comment:
why is this line needed here

Reply:
Thanks for catching this. This is now not required since all the compiler warnings are resolved.